### PR TITLE
Update profile page logic for new data

### PR DESCRIPTION
**Pull request by tiagojsag**
https://github.com/sei-international/TRASE/blob/new-data-2017/DB/mapping_variables

some quants/inds were renamed, others moved around, so it's api changing time!

### CODE REVIEW

**Comment by agnessa** [comment]
🤦‍♀️

File: app/models/ind.rb [hunk 1]
```diff
@@ -30,6 +30,6 @@ class Ind < ActiveRecord::Base
     has_many :context_indicators, as: :indicator
 
     scope :place_temporal, -> {
-      where("place_factsheet AND place_factsheet_temporal OR name IN ('POTENTIAL_SOY_RELATED_DEFOR_ind', 'TOTAL_DEFOR_RATE')")
+      where('place_factsheet AND place_factsheet_temporal')
```
Comment (agnessa) [inline app/models/ind.rb]: Are you sure `POTENTIAL_SOY_RELATED_DEFOR_ind` is no longer required here? This is clearly not a part of code I've written that I'm particularly proud of, but while I understand that `TOTAL_DEFOR_RATE` not only changed names but also changed from `ind` to `qual`, whereas `POTENTIAL_SOY_RELATED_DEFOR_ind` is still an ind and maybe relevant here? :D

Comment (tiagojsag) [reply app/models/ind.rb]: POTENTIAL_SOY_RELATED_DEFOR_ind is gone from the new db entirely

Comment (agnessa) [reply app/models/ind.rb]: oh in that case maybe it needs removing also from `place_attributes.rb`?

Comment (tiagojsag) [reply app/models/ind.rb]: seems I missed that and a couple more things :s

**Pull request merged**